wagtail_guide/wagtail_hooks.py

```diff
@@ -20,14 +23,36 @@ def register_admin_urls():
     ]
 
 
-@hooks.register("register_admin_menu_item")
-def register_editor_guide_menu_item():
-    return MenuItem(
-        _("Editor guide"),
-        reverse("wagtaileditorguide"),
-        classnames="icon icon-help",
-        order=1000,
-    )
+if WAGTAIL_VERSION >= (4, 1) and wagtail_guide_settings.ADD_WAGTAIL_GUIDE_TO_HELP_MENU:
+
+    @hooks.register("register_help_menu_item")
+    def register_editor_guide_menu_item():
+        return MenuItem(
+            _(WAGTAIL_GUIDE_MENU_LABEL),
+            reverse("wagtaileditorguide"),
+            classnames="icon icon-help",
+            order=1000,
+        )
+
+else:
+
+    @hooks.register("register_admin_menu_item")
+    def register_editor_guide_menu_item():
+        return MenuItem(
+            _(WAGTAIL_GUIDE_MENU_LABEL),
+            reverse("wagtaileditorguide"),
+            classnames="icon icon-help",
+            order=1000,
+        )
```

Comment:
There is some duplicate code here, not sure how to conditional change a hook on it's own...